src/DataFrames.jl

```diff
@@ -1,6 +1,10 @@
 __precompile__(true)
 module DataFrames
 
+if !isdefined(Base, :pairs)
+    pairs(x) = x
```

Comment:
This seems like a dangerous definition to have lying around, but I guess it's okay until we stop supporting 0.6.

Reply:
Yeah. Shouldn't Compat provide something like that, since many packages will have the same problem?

BTW, is Femtocleaner able to remove this kind of `isdefined` check? Or should we use `VERSION` checks instead?

Reply:
I'm not sure about FemtoCleaner, but `pairs` will be available in Compat as of 0.40.0: https://github.com/JuliaLang/METADATA.jl/pull/12530

Reply:
So do we want to use `Compat.pairs` here rather than a custom method?

Reply:
I'd say so, now that it's available in Compat. Just have to bump the REQUIRE to 0.40.0.

Reply:
now we might as well wait till Compat gets Unicode since DataFrames show totally breaks w/o it.

Reply:
Nah, I'd say just add
```julia
if VERSION >= v"0.7.0-DEV.2915"
    using Unicode
end
```